packages/syft/src/syft/lib/python/slice.py

```diff
@@ -26,6 +32,15 @@ def __init__(
         step: Optional[Any] = None,
         id: Optional[UID] = None,
     ):
+        """
+        This class will receive start, stop, step and ID as valid parameters.
+
+        :param start: Index/position where the slicing of the object starts.
```

Comment:
MAYBE: could we switch to `Google Python` Docstring for consistency in the codebase.